Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -3194,7 +3194,7 @@ void GraphicsContextGLANGLE::paintCompositedResultsToCanvas(ImageBuffer& imageBu
     });
 }
 
-void GraphicsContextGLANGLE::withDrawingBufferAsNativeImage(std::function<void(NativeImage&)> func)
+void GraphicsContextGLANGLE::withDrawingBufferAsNativeImage(Function<void(NativeImage&)> func)
```

Comment:
Maybe we should have caught this in the original patch?

Reply:
Them sloppy reviewers..